collections/nemo_nlp/nemo_nlp/text_data_utils.py

```diff
@@ -619,17 +623,104 @@ def process_wkt(data_dir):
     return data_dir
 
 
+def download_wkt2(data_dir):
+    os.makedirs('data/lm', exist_ok=True)
+    logger.warning(f'Data not found at {data_dir}. '
```

Comment:
Does this work without the logger from ExpManager?

Reply:
yes